tbapy/main.py

```diff
@@ -387,3 +408,102 @@ def district_teams(self, district, simple=False, keys=False):
             return self._get('district/%s/teams/keys' % district)
         else:
             return [Team(raw) for raw in self._get('district/%s/teams' % district)]
+
+    def update_trusted(self, auth_id, auth_secret, auth_event_key):
+        """
+        Set Trusted API ID and Secret and the event key they are assigned to.
+
+        :param auth_id: Your event authorization ID, obtainable at https://www.thebluealliance.com/request/apiwrite
+        :param auth_secret: Your event authorization secret, obtainable at https://www.thebluealliance.com/request/apiwrite
+        """
+        self.auth_id = auth_id
+        self.auth_secret = auth_secret
+        self.auth_event_key = auth_event_key
+
+    def update_event_info(self, data):
+        """
+        Update an event's info on The Blue Alliance.
```

Comment:
I went with that because of how TBA's URL which is event/%s/info/update. All of the functions take the last two tokens in the URL, switch them around and put "event" in between. I didn't put too much thought into it cause I expected you to want to change them.